app/livechat/server/lib/Livechat.js

```diff
@@ -766,15 +641,24 @@ export const Livechat = {
 	saveDepartment(_id, departmentData, departmentAgents) {
 		check(_id, Match.Maybe(String));
 
-		check(departmentData, {
+		const defaultValidations = {
 			enabled: Boolean,
 			name: String,
 			description: Match.Optional(String),
 			showOnRegistration: Boolean,
 			email: String,
 			showOnOfflineForm: Boolean,
+		};
+
+		// The Livechat Form department support addition/custom fields, so those fields need to be added before validating
+		Object.keys(departmentData).forEach((field) => {
+			if (!Object.keys(defaultValidations).some((v) => v === field)) {
```

Comment:
no need to iterate over all keys all the time, this is better to check if a key already exists:
```suggestion
			if (!defaultValidations.hasOwnProperty(field)) {
```